x/init.go

```diff
@@ -42,45 +39,13 @@ var (
 	lastCommitTime string
 )
 
-// SetTestRun sets a variable to indicate that the current execution is a test.
-func SetTestRun() {
```

Comment:
This looks ok to remove.
It is related to /root/dev/dgraph/worker/predicate_test.go at the line 124
The test was commented out by Martin with some TODOs